src/components/ChatEntry.jsx

```diff
@@ -1,20 +1,44 @@
 import './ChatEntry.css';
+import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+
+const ChatEntry = ({id, sender, body, timeStamp,liked, onLikeToggle}) => {
```

Comment:
```suggestion
const ChatEntry = ({ id, sender, body, timeStamp,liked, onLikeToggle }) => {
```

Spaces around curly braces when destructuring